tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Remoting/Client/HttpProcessMessageIntegration.cs

```diff
@@ -0,0 +1,96 @@
+// <copyright file="HttpProcessMessageIntegration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if NETFRAMEWORK
+using System;
+using System.ComponentModel;
+using System.IO;
+using System.Runtime.Remoting.Channels;
+using System.Runtime.Remoting.Messaging;
+using Datadog.Trace.ClrProfiler.AutoInstrumentation.Http.WebRequest;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Logging;
+using Datadog.Trace.Propagators;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Remoting.Client
+{
+    /// <summary>
+    /// System.Runtime.Remoting.Channels.IClientChannelSink.ProcessMessage calltarget instrumentation
+    /// </summary>
+    [InstrumentMethod(
+        AssemblyName = "System.Runtime.Remoting",
+        TypeName = "System.Runtime.Remoting.Channels.Http.HttpClientTransportSink",
+        MethodName = "ProcessMessage",
+        ReturnTypeName = ClrNames.Void,
+        ParameterTypeNames = new[] { "System.Runtime.Remoting.Messaging.IMessage", "System.Runtime.Remoting.Channels.ITransportHeaders", "System.IO.Stream", "System.Runtime.Remoting.Channels.ITransportHeaders&", "System.IO.Stream&", },
+        MinimumVersion = RemotingIntegration.Major4,
+        MaximumVersion = RemotingIntegration.Major4,
+        IntegrationName = RemotingIntegration.IntegrationName)]
+    [Browsable(false)]
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    // ReSharper disable once InconsistentNaming
+    public class HttpProcessMessageIntegration
+    {
+        internal const IntegrationId WebRequestIntegrationId = Configuration.IntegrationId.WebRequest;
```

Comment:
nit: this appears in each integration but doesn't seem to be used?

Reply:
Fixed in 424990f39cd3c504c09b7978f938c653e225a365